src/profile-logic/import/dhat.js

```diff
@@ -191,6 +191,33 @@ export function attemptToConvertDhat(json: mixed): Profile | null {
   const otherCategory = 0;
   const otherSubCategory = 0;
 
+  // Insert a root function that is the command that was run. Note that dhat-rs includes
+  // a [root] frame, while the Valgrind implementation does not.
+  funcTable.name.push(stringTable.indexForString(dhat.cmd));
+  funcTable.isJS.push(false);
+  funcTable.relevantForJS.push(false);
+  funcTable.resource.push(-1);
+  funcTable.fileName.push(null);
+  funcTable.lineNumber.push(null);
+  funcTable.columnNumber.push(null);
+  const rootFuncIndex = funcTable.length++;
+
+  frameTable.address.push(-1);
+  frameTable.line.push(null);
+  frameTable.column.push(null);
+  frameTable.category.push(otherCategory);
+  frameTable.subcategory.push(otherSubCategory);
+  frameTable.innerWindowID.push(null);
+  frameTable.implementation.push(null);
+  frameTable.func.push(rootFuncIndex);
```

Comment:
I wonder if `nativeSymbol` and `inlineDepth` should be populated as well (respectively with `null`s and `0` according to what we do in process-profile.js and other places). I'd say yes :-)
And probably also below when populating the `frameTable`.